src/mlpack/bindings/python/print_pyx.cpp

```diff
@@ -186,10 +186,14 @@ void PrintPYX(const ProgramDoc& programInfo,
       << endl;
 
   // Determine whether or not we need to copy parameters.
-  cout << "  if copy_all_inputs:" << endl;
-  cout << "    SetParam[bool](<const string> 'copy_all_inputs', "
+  cout << "  if isinstance(copy_all_inputs, bool):" << endl;
+  cout << "    if copy_all_inputs:" << endl;
+  cout << "      SetParam[bl](<const string> 'copy_all_inputs', "
       << "copy_all_inputs)" << endl;
-  cout << "    CLI.SetPassed(<const string> 'copy_all_inputs')" << endl;
+  cout << "      CLI.SetPassed(<const string> 'copy_all_inputs')" << endl;
+  cout << "  else:" << endl;
+  cout << "    raise TypeError(" <<"\"copy_all_inputs must have type 'bool'!\""
```

Comment:
Can you fix the style here?  `<< "\'copy_all_inputs\' must have type 'bool'!\""`